modules/installation-configuration-parameters.adoc

```diff
@@ -1007,6 +1007,12 @@ For a private cluster, specify a private subnet for each availability zone.
 For clusters that use AWS Local Zones, you must add AWS Local Zone subnets to this list to ensure edge machine pool creation.
 |Valid subnet IDs.
 
+|platform:
+  aws:
+    publicIpv4Pool:
+|IPv4 pool name from the public IPv4 pools in the AWS account to configure the cluster with. Your AWS account must be provisioned for public IPv4 pool. You must have 3n+1 pools available where n is the total number of AWS zones used to deploy the cluster.
```

Comment:
```suggestion
|Public IPv4 Pool ID used to allocate Elastic IPs when `publish` is set to `External`. The pool must be provisioned and advertised in the same AWS account and region of the cluster. You must have (2*n + 1) IPv4 available in the pool where `n` is the total number of AWS zones used to deploy the Network Load Balancer for API, NAT Gateways, and Bootstrap node. For more information about AWS BYO IPv4, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-byoip.html#byoip-onboard[Onboard your BYOIP]
```

I suggest to keep the AWS doc reference to get users aware of AWS BYOIPv4 feature (it is a common mistake when talking about BYO IP because it can be IPAM, CoIP, etc).

Reply:
and not 3n+1 @mtulio can you confirm that it's 2n+1 as started [here](https://github.com/openshift/installer/blob/master/docs/user/aws/custom_public_ipv4.md)?